roles/installer/ospd/images/build/tasks/main.yml

```diff
@@ -1,2 +1,29 @@
 ---
-# TODO: Add images build code here
+- fail: msg="Check is image build path is defined"
+  when: image_build_path is not defined
+
+- name: Create overcloud_images directory
+  file: path={{ image_build_path }} state=directory
+
+- name: Download guest image for build
+  get_url:
+      dest: "{{ image_build_path }}/guest_image.qcow2"
+      url: "{{ installer.images.overcloud.guest_image }}"
+
+- name: Build OC images
+  shell: |
+      pushd ~/overcloud_images
+      openstack overcloud image build --all > openstack-build-images.log
+  environment:
+      DIB_LOCAL_IMAGE: "{{ image_build_path }}/guest_image.qcow2"
+      DIB_YUM_REPO_CONF: "/etc/yum.repos.d/rhos-release-{{ product.version.major }}-director.repo /etc/yum.repos.d/rhos-release-{{ product.version.major }}.repo /etc/yum.repos.d/rhos-release-rhel-{{ distro.full_version }}.repo"
+      RHOS: 1
+      RUN_RHOS_RELEASE: 1
+      RHOS_RELEASE: "{{ product.version.major }}_director"
+      USE_DELOREAN_TRUNK: 0
+# TODO: Add portal credentials and repositories - not working so far - invalid credentials
```

Comment:
yes, when we need it, we will add them back :)
If you want to keep it for reference, please create a local git branch and commit it there, this does not belong in production.